apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/DbUtils.java

```diff
@@ -40,13 +48,128 @@ public static void close(AutoCloseable... closeables) {
         }
     }
 
+    /**
+     * Close a result set without throwing an exception.
+     *
+     * @param rs the result set or null
+     */
+    public static void closeSilently(ResultSet rs) {
+        if (rs != null) {
+            try {
+                rs.close();
+            } catch (SQLException ignore) {}
+        }
+    }
+
+    /**
+     * Close a reader without throwing an exception.
+     *
+     * @param reader the reader or null
+     */
+    public static void closeSilently(Reader reader) {
+        if (reader != null) {
+            try {
+                reader.close();
+            } catch (Exception e) {
+                // ignore
+            }
+        }
+    }
+
+    /**
+     * Close a writer without throwing an exception.
+     *
+     * @param writer the writer or null
+     */
+    public static void closeSilently(Writer writer) {
+        if (writer != null) {
+            try {
+                writer.flush();
+                writer.close();
+            } catch (Exception e) {
+                // ignore
+            }
+        }
+    }
+
+    /**
+     * Create an output stream to write into the file.
+     * This method is similar to Java 7
+     * <code>java.nio.file.Path.newOutputStream</code>.
+     *
+     * @param fileName the file name
+     * @param append if true, the file will grow, if false, the file will be
+     *            truncated first
+     * @return the output stream
+     */
+    public static OutputStream newOutputStream(String fileName, boolean append)
+            throws IOException {
+        Objects.requireNonNull(fileName, "fileName is NULL");
+        FilePath filePath = FilePath.get(fileName);
```

Comment:
Do we actually require FilePath from H2?

Reply:
Removed completely. Path is taken from configs.